src/modules/three-d-structures/ThreeDStructuresPage.js

```diff
@@ -56,27 +54,9 @@ const ThreeDStructuresPage = () => {
   };
 
   const render3dStructure = (threeDStructure) => {
-    const threeDStructureTitle = get3dStructureTitle(threeDStructure);
-    const threeDStructureFieldsText = Object.entries(threeDStructure).reduce((acc, [key, value]) => {
-      return key === 'id' ? acc
-        : (acc === '' ? '' : acc + '\n')
-        + toTitleCase(useForm.getLabel(key, ['_3d_structures', threeDStructure.feature_type])) + ': '
-        + useForm.getLabels(value, ['_3d_structures', threeDStructure.feature_type]);
-    }, '');
     return (
-      <ListItem
-        containerStyle={commonStyles.listItem}
-        key={threeDStructure.id}
-        onPress={() => edit3dStructure(threeDStructure)}
-      >
-        <ListItem.Content style={{overflow: 'hidden'}}>
-          <ListItem.Title style={commonStyles.listItemTitle}>{threeDStructureTitle}</ListItem.Title>
-          {threeDStructureFieldsText !== '' && (
-            <ListItem.Subtitle>{threeDStructureFieldsText}</ListItem.Subtitle>
-          )}
-        </ListItem.Content>
-        <ListItem.Chevron/>
-      </ListItem>
+      <ThreeDStructureItem item={threeDStructure} edit3dStructure={(item) => edit3dStructure((item))}
```

Comment:
This could be condensed if it all fits on one line. Might not even need the return (

const render3dStructure = (threeDStructure) => <ThreeDStructureItem item={threeDStructure} edit3dStructure={(item) => edit3dStructure((item))}/>

Reply:
desnt fit in 1 line.